modules/github-oidc-role/main.tf

```
@@ -0,0 +1,54 @@
+locals {
+  enabled             = module.this.enabled
+  aws_canned_policies = [for arn in var.aws_iam_policies : arn if can(regex("^arn:aws:iam::aws:policy/", arn))]
+  aws_policies        = length(local.aws_canned_policies) > 0 ? local.aws_canned_policies : null
```

The "aws" prefix is redundant, and given that the policies can be our custom names, somewhat misleading. Also, as long as you are using a regex, you might as well allow for other partitions.

```suggestion
  canned_policies = [for arn in var.iam_policies : arn if can(regex("^arn:aws[^:]*:iam::aws:policy/", arn))]
  policies        = length(local.canned_policies) > 0 ? local.canned_policies : null
```

If you prefer, you can replace "canned" with "aws_managed", which aligns with the AWS documentation.